packages/web-components/src/components/tag/dismissible-tag.ts

```diff
@@ -169,13 +171,13 @@ class CDSDismissibleTag extends HostListenerMixin(FocusMixin(CDSTag)) {
           <button
             class="sb-tooltip-trigger"
             role="button"
-            aria-labelledby="content"
+            aria-label="${dismissActionLabel}"
```

Comment:
Is aria-label what we typically do here or could this remain `aria-labelledby="content"` since `dismissActionLabel` is rendered in the `#content`?